pkg/internalapis/apis/cstor/cstorpool_topology.go

```diff
@@ -66,6 +66,63 @@ const (
 	PoolScanFuncStates
 )
 
+// VdevState represent various device/disk states
```

Comment:
What do vdev states represent here?

Reply:
As mentioned it represents the state of device in pool